roles/glance/templates/etc/serverspec/glance_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+[['glance-api.conf', 640], ['glance-api-paste.ini', 640], ['glance-registry.conf', 640], ['glance-registry-paste.ini', 640], ['glance-swift-store.conf', 640], ['policy.json', 644]].each do |file| 
+  describe file("/etc/glance/#{file[0]}") do
+    it { should be_mode file[1] }
```

Comment:
Why is the file / mode set done differently here vs the other roles? This one is a list of lists, the others were different (e.g. https://github.com/blueboxgroup/ursula/pull/1593/files#diff-30e9feac5304c5dac9cf568e5747c52eR3 )